config/env.development

```diff
@@ -62,7 +62,7 @@ SECRET=secret-sauce
 ADMINISTRATORS=user1@example.com
 
 # Origins of web apps that we'll allow for redirects. See src/api/auth/test
-ALLOWED_APP_ORIGINS=http://localhost:8000 http://localhost:8888
+ALLOWED_APP_ORIGINS=http://localhost:8000 http://localhost:10000
```

Comment:
You won't need to do this.  the `ALLOWED_APP_ORIGINS` is for the auth service to know about front-end origins that are OK to redirect a token back to.  You won't have any interaction with the front-end at all.

Reply:
So sorry about this, I don't think this was supposed to be a change here.